src/test/analyzer_test.ts

```diff
@@ -12,15 +12,17 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-'use strict';
+/// <reference path="../../node_modules/@types/mocha/index.d.ts" />
```

Comment:
I know I was surprised too. Only one reference is needed since it's treated as a global, but since there's no one entry-point to our tests suite adding it to all seemed safest in the event that files are moved around and/or refactored.

(ex: https://github.com/Polymer/polymer-analyzer/blob/master/src/test/analyzer_test.ts)